### ISSUE TITLE
[1.3.20] Error when pulling SSS function

### ISSUE THREAD

**Issue by antho1404** [graphcool/graphcool-cli#191]
I'm not sure this is related to the CLI or the backend but I have an issue when I pull my SSS functions (with `graphcool init` -> `Copying an existing project`)

```
The supplied query for the server side subscription `event/created/trigger-service` is invalid
```

with the following subscription query

```
subscription {
  Event(filter: {
    mutation_in: CREATED
  }) {
    updatedFields
    node {
      trigger {
        id
      }
    }
  }
}
```

It works when I don't have any nested on my subscription. When I create an Event (in the case) everything works well and I have access to all my data

Version
`graphcool/1.3.20 (darwin-x64) node-v8.4.0`

**Comment by marktani** [comment]
Might be related to this problem: https://github.com/graphcool/graphcool/issues/465
Did you change the relation from `Event` to `Trigger`?

**Comment by kbrandwijk** [comment]
I highly doubt if that's related, because doing `gc init` with `copy existing project` has no local 'future' changes to the Types.

**Comment by antho1404** [comment]
@marktani I did rename `Trigger` before but not `Event` to `Trigger`.
I've tried to check in console.graph.cool and I have some error when I open the chrome console `Error: Couldn't generate SSS example event` so I think like @kbrandwijk sais this is not related to the CLI but maybe more to the backend

Everything works good, the function is triggered with all the data but just it seems like the validation fails maybe.

Is it possible to define a SSS and fetch a relation or should I do that in my function/webhook this will probably solve the problem but if I can avoid to fetch some data that I can have from the event in my function that would be better.

Maybe this ticket should be moved to the correct project https://github.com/graphcool/graphcool/ i guess
